src/BinSkim.Driver/RoslynAnalyzer/ILImporter.Driver.cs

```diff
@@ -0,0 +1,915 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// Source: https://github.com/dotnet/corert/blob/master/src/Common/src/TypeSystem/IL/ILImporter.cs
+
+using System;
+
+namespace Microsoft.CodeAnalysis.IL
+{
+    //
+    // Corresponds to "I.12.3.2.1 The evaluation stack" in ECMA spec
+    //
+    internal enum StackValueKind
```

Comment:
separate file for this? perhaps you aren't looking for this level of feedback yet?? tell me if so